packages/design-system/src/components/web-components/ds-drawer/ds-drawer.tsx

```diff
@@ -0,0 +1,89 @@
+import { define } from '../preactement/define';
+import { Drawer } from '../../Drawer';
+import { DrawerProps } from '../../Drawer/Drawer';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'aria-label',
```

Comment:
`aria-label` here is duplicated in the parent ds-drawer element and in the child button element. I think you need to alias this like we do for the `root-id` attribute.

Reply:
Hey @jack-ryan-nava-pbc, thanks for pointing that out! I think I might be seeing a different issue than the duplication you mentioned. When I pass an aria-label value to the ds-drawer parent in storybook, I expect it to be applied to the [close button element](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/components/Drawer/Drawer.tsx#L123). Instead, the close button element's aria-label is defaulting to "Close Help Drawer" because the `ariaLabel` prop appears to be `undefined`. 

I think the setup might need an adjustment, but I'm curious—could you outline the steps you took to reproduce the parent/child aria-label duplication?

Reply:
Sure! So the easiest way to replicate this is in storybook itself. 
1. Set the `aria-label` value in the controls section to something
2. Check that it is set on the `ds-drawer` parent element and the button element in the `ds-c-drawer__header`

You can also do this by setting the value in the `formattedArgs` hash in the stories file. I set the `aria-label` key to be 'Test label' and saw it duplicated.

Reply:
@jack-ryan-nava-pbc It seems I tested this earlier while I had some unintended local changes applied. Now I'm seeing the duplication. Do you think we should take a similar approach to the one taken in [ds-tooltip](https://github.com/CMSgov/design-system/pull/3249#discussion_r1781963532)?

Reply:
I think that would make sense so we avoid any confusion around the labelling as read by screen readers.